artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPLargeMessage.java

```diff
@@ -130,6 +130,10 @@ private AMQPLargeMessage(final AMQPLargeMessage copy,
                                   final SequentialFile fileCopy,
                                   final long newID) {
       super(copy);
+      if (this.messageAnnotations != null) {
+         this.messageAnnotations.getValue().forEach((a, b) -> logger.info("{}={}", a, b));
+      }
+      this.messageAnnotations = null;
```

Comment:
I'm confused....why is this nulling the messageAnnotations that the super constructor just copied using the newly added filtering bit?

Reply:
I'm an idiot and I forgot to remove it.
although the copy is not fixing it... :/

Reply:
fixed it... thanks a lot @tabish121